lib/http.js

```diff
@@ -273,22 +275,25 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
     routes.automatedIssue,
     // TODO: validate body
     asyncHandler(async (req, res) => {
-      const {
-        body: {credential},
-        headers: {authorization},
-      } = req;
-
-      const tokenStartIndex = 7;
-      if(!(authorization && typeof authorization === 'string' &&
-        authorization.startsWith('Bearer ') &&
-        authorization.length > tokenStartIndex)) {
-        throw new BedrockError('Permission denied.', 'NotAllowedError', {
-          httpStatusCode: 400,
-          public: true,
-        });
+      let token;
+      try {
+        // parse throws if the authorization header is missing
+        const auth = authorization.parse(req.get('authorization'));
+        if(!(auth.scheme === 'Bearer' && auth.token)) {
```

Comment:
This is not a problem, just a note for you for the future: the `authorization` header is also used for http signatures, and at some point we'll likely enable this endpoint for use by frontends that use zcaps for authz. This is also probably a good reason for using a parser so we can check for multiple valid schemes.